### DISCUSSION TITLE
custom skin

### DISCUSSION THREAD

**Discussion by jessvoo** [community/community#18914] [Other Feature Feedback, Questions, & Ideas]

**Comment by MathiasReker** [comment]
Hello!

**Reply by jessvoo** [reply]
hello Mathias,
 
When I make a film selection, I want it to go directly to information
 
best
 
RG

**Reply by jessvoo** [reply]
skin.arctic.zephyr.2  that's what it's abou

**Reply by jessvoo** [reply]
like this 
https://streamable.com/x3n65j

**Reply by jessvoo** [reply]
https://streamable.com/1pj8vy

**Comment by Vishal-beep136** [comment]
custom skin! yeah

**Reply by jessvoo** [reply]
hello 
When I make a film selection, I want it to go directly to information.

 
best
 
RG

**Reply by jessvoo** [reply]
https://streamable.com/1pj8vy

**Reply by jessvoo** [reply]
like this
https://streamable.com/x3n65j
